src/lock_file/satisfiability.rs

```diff
@@ -1199,4 +1204,54 @@ mod tests {
         // This should satisfy:
         assert!(pypi_satifisfies_requirement(&spec, &locked_data));
     }
+
+    // Test if the lockfile satisfies when pypi-dependencies are defined but all overwritten by conda deps.
+    #[test]
+    fn test_satisfiability_with_pypi_deps_no_pypi_install() {
```

Comment:
We have tests location for these kind of tests. Look in test-data/satisiability

Reply:
Thanks, overlooked that!